test/models/name_test.rb

```diff
@@ -2639,6 +2643,32 @@ def test_suggest_alternate_spelling
                            Name.suggest_alternate_spellings("Lecanoa grandis"))
   end
 
+  def test_synonym_ids
```

Comment:
A convenient place to add coverage to fix Coveralls failure.
Coveralls failed in Commit #8c963439 although there were no newly uncovered lines.
So I picked some existing uncovered lines (though not related to the subject of this PR) and covered them.